wp-content/themes/bpr2018/resources/templates/category.tpl.php

```diff
@@ -76,14 +76,16 @@ class="img-40 lazy-image"
                     </p>
       
                     <div class="post-author post-date font-size-20">
-                      <a href="<?php echo esc_url(get_author_posts_url(get_the_author_meta('ID'))); ?>">
-                        <?php the_author(); ?>
-                      </a>
+                      <span itemprop="author" hidden aria-hidden><?php the_author(); ?></span>
                       <?php
-                      if (get_the_date()) { echo ' | <time>' . get_the_date() . '</time>'; }
+                      if (function_exists('coauthors_posts_links')) {
+                        coauthors_posts_links(',',',',null,null,true);
+                      } else {
+                        the_author_posts_link();
+                      } 
+                       if (get_the_data()) { echo ' | <time itemprop="datePublished">' . get_the_date() . '</time>'; }
```

Comment:
It seems that we've lost the `get_the_date` call; maybe replace `get_the_data` with `get_the_date`?

```php
if (get_the_date()) { echo ' | <time>' . get_the_date() . '</time>'; }
```

Reply:
cool, on it

Reply:
Just pushed